.github/workflows/reg.yml

```diff
@@ -0,0 +1,49 @@
+name: Visual Regression Testing
+
+on: [push]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+#   strategy:
+#     matrix:
+#       node-version: [8.x, 10.x, 12.x]
+
+    steps:
+    - uses: actions/checkout@v3
+      with:
+        fetch-depth: 0
+    - name: Use Node.js v16
+      uses: actions/setup-node@v3
+      with:
+        node-version: "16.x"
+    - name: npm install, build, and test
+      run: |
+        npm install --pure-lockfile
+        npx lerna bootstrap --no-ci
+      env:
+        CI: true
+    - name: workaround for detached HEAD
```

Comment:
This would be good to include a comment linking to https://github.com/reg-viz/reg-suit#workaround-for-detached-head

Reply:
added as a comment